LeapEngine/Utils/Physics.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <vec3.hpp>
+
+namespace leap
+{
+	class Collider;
+
+	struct RaycastHitInfo final
+	{
+		Collider* pCollider{};
```

Comment:
Add rigidbody value:
The Rigidbody of the collider that was hit. If the collider is not attached to a rigidbody then it is null.